test/acceptance/test/smoke_tests.go

```diff
@@ -26,10 +26,16 @@ func FileExists(name string) bool {
 	return true
 }
 
-var _ = Describe("WEGO Acceptance Tests ", func() {
+var _ = Describe("WEGO Acceptance Tests", func() {
+
+	var session *gexec.Session
+	var err error
 
 	BeforeEach(func() {
 
+		Context("Given I have a wego binary installed on my local machine", func() {
+			Expect(FileExists(WEGO_BIN_PATH)).To(BeTrue())
+		})
```

Comment:
I think Context container is not recommended to be used in BeforeEach or It blocks, instead it is preferred to keep them outside. Any particular reason for using it in this way?

Reply:
It was the only way I could make the script run... I wanted to run the 'Given' condition before each test case but the hierarchy of 'Describe', 'Context', 'It', 'By' was giving me problems

Reply:
I've changed `Context` to `By` — seems to work just fine.